CHANGELOG.md

```diff
@@ -1,32 +1,112 @@
+  - New `Botonic.clearMessages()` method to force the deletion of the messages stored in the webchat.
+
+- [@botonic/plugin-contentful](https://www.npmjs.com/package/@botonic/plugin-contentful)
+
+  - Improve tokenization of words containing hyphens and number (eg. covid-19).
+  - Improve tokenization of "pronoms febles" in Catalan.
+  - New API to update the texts of contentful contents.
+  - Tool for translators to translate the content texts through CSV files.
+  - Now buttons can refer to contentful queues (or any other top content).
+  - Now all Content's have the "id" field.
+
+### Changed
+
+- [Project](https://github.com/hubtype/botonic)
+
+  - Dependabot to check for updates on a weekly basis.
+  - Dev dependencies are now installed in the repository top folder (parent config).
+- [@botonic/cli](https://www.npmjs.com/package/@botonic/cli)
+  - **Breaking change**: The Minimal Node version is `node-v10.19.0`
+  - Webpack's resolve configuration is now unified into `resolveConfig` object.
+
+- [@botonic/react](https://www.npmjs.com/package/@botonic/react)
+
+  - **Breaking change**: `emojiPicker` property is renamed `enableEmojiPicker`.
+  - All styles of the project are converted into [Styled Components](https://styled-components.com/).
+
+- [@botonic/plugin-contentful](https://www.npmjs.com/package/@botonic/plugin-contentful)
+  - Upgrade to typescript 3.8
+  - **Breaking change**: Refactor TopContent and Content base classes. ModelType renamed to TopContentType.
+  - **Breaking change**: Refactor so that SearchResult now returns ContentId's instead of ContentCallback's.
+  - Now only the required language-specific [NLP.js](https://www.npmjs.com/package/node-nlp) folders are imported. This enables tree shaking to exclude big sentiment analysis files.
+  - Error reporting decorator now also reports the originating exception.
+  - Flag to disable the CMS delivery cache.
+- [@botonic/plugin-dynamodb](https://www.npmjs.com/package/@botonic/plugin-dynamodb)
+  - Upgrade to typescript 3.8
+  - **Breaking change**: Move Env to its own file to ease tree-shaking
+
+### Fixed
+
+- [@botonic/react](https://www.npmjs.com/package/@botonic/react)
+  - Noto Sans JP now does not show up as default typography.
+  - Deprecated `simplebar` warnings have been removed.
+  - Fixed outer page scrolling that produced unstable behavior.
+  - Fixed noopener to avoid security risk in location links.
+  - Fixed `compromise` import in `@botonic/cli`'s `nlu` template.
+  - Webviews are now rendered within `div` instead of `iframe` for a better visualization when developing locally without affecting production builds
+
 ## [0.10.1] - 2020-14-01
+
 ### Added
-- Added default config for every template `{  defaultDelay: 0.4,  defaultTyping: 0.6  }` to avoid extravagant warnings in `botonic serve`.
+
+- Added default config for every template `{ defaultDelay: 0.4, defaultTyping: 0.6 }` to avoid extravagant warnings in `botonic serve`.
```

Comment:
"extravagant" sounds weird for a technical doc. maybe "confusing", "misleading"?